go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/cosmos/gogoproto v1.4.11
 	github.com/cosmos/ledger-cosmos-go v0.13.3
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.2.0
-	github.com/ethereum/go-ethereum v1.12.0
+	github.com/ethereum/go-ethereum v1.13.15
```

Comment:
Upgraded versions according to output of go vulnerability [check](https://github.com/dydxprotocol/cosmos-sdk/actions/runs/9966208844/job/27537901390)